index.html

```diff
@@ -1,9 +1,11 @@
+<!DOCTYPE html>
 <html>
   <head>
     <title>Kanji Sheets</title>
-    <link href="/css/main.css" rel="stylesheet" type="text/css" />
+    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+    <link href="css/main.css" rel="stylesheet" type="text/css" />
```

Comment:
I kept all of these paths a certain way because when I redeploy my github page it will conflict with other repositories I have deployed that also have a file of css/main.css. 

It will reference this page's css file: http://jensechu.github.io/